src/config/kernel.md

```diff
@@ -21,28 +21,40 @@ modules.
 
 ## Removing old kernels
 
-To list kernels that can be purged
+When updating the kernel, old versions are left behind in case it is neccessary to roll back to an older version.
+Over time, many old kernel version accumulate and make updating dmks modules take a long time.
```

Comment:
This could be shortened to something just like 'As old kernels accumulate, you may wish to remove versions that are no longer necessary.'